src/components/versionSelect.tsx

```diff
@@ -1,17 +1,21 @@
-import config from "@payload-config";
-import { getPayload } from "payload";
-import { VersionSelectUi } from "./versionSelect-ui";
+import config from '@payload-config';
+import { getPayload } from 'payload';
+import { VersionSelectUi } from './versionSelect-ui';
 
 export async function VersionSelect() {
   const payload = await getPayload({ config });
   const versions = await payload.find({
-    collection: "versions",
+    collection: 'versions',
     where: {
       isVisible: { equals: true },
     },
-    sort: "-slug",
+    sort: '-slug',
+    locale: 'sk',
```

Comment:
Nenajde tohle query jen `sk` verze? 🤔 Ale chapu ze tohle je jen beginning :)

Reply:
Respektive všechny verze, které mají locale `sk`, tzn jsou prelozene do slovenstiny? Tim padem to nenajde `cz` verze